velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -44,6 +44,11 @@ std::string makeCreateTableSql(
     } else if (child->isMap()) {
       sql << "MAP(" << child->asMap().keyType()->kindName() << ", "
           << child->asMap().valueType()->kindName() << ")";
+    } else if (child->isShortDecimal() || child->isLongDecimal()) {
+      int precision;
+      int scale;
+      getDecimalPrecisionScale(*child, precision, scale);
```

Comment:
nit: Would be nice to change the signature of this function to return a pair of precision and scale.

```
auto [precision, scale] = getDecimalPrecisionScale(*child);
```

Reply:
The decimal add PR has this change. I will rebase after that change.